src/js/msp/MSPHelper.js

```diff
@@ -423,6 +423,11 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 if (semver.gte(CONFIG.apiVersion, "1.34.0")) {
                     GPS_CONFIG.auto_config = data.readU8();
                     GPS_CONFIG.auto_baud = data.readU8();
+
+                    if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+                        GPS_CONFIG.home_point_once = data.readU8() !== 0;
```

Comment:
Looks like I overlooked this - this should not modify the data that is received.

Reply:
I will push a new PR tomorrow fixing that.